src/App.css

```diff
@@ -331,8 +335,8 @@ footer {
 
 @media only screen and (min-width: 1024px) {
   .logo-large-placement {
-      left: -97%;
-      position: relative;
-      top: 5rem;
+    left: -100%;
```

Comment:
Were all the pages checked to see how this would affect the logo?

Reply:
Just checked it out how it would look on landing, approved. However I am wondering if a different class should be added to the logo on other screens to help fix their positioning?

Reply:
<img width="1092" alt="Screenshot 2019-12-17 at 10 21 49" src="https://user-images.githubusercontent.com/13155310/70987092-23cd9100-20b7-11ea-86ff-77376f50a185.png">